onert-micro/luci-interpreter/include/luci_interpreter/core/Tensor.h

```diff
@@ -133,6 +133,27 @@ class RuntimeShape
 class Tensor
 {
 public:
+  static const float max_value(const circle::Tensor *circle_tensor)
+  {
+    const auto *quant_params = circle_tensor->quantization();
+    if (quant_params == nullptr)
+    {
+      assert(false && "There is no quantization params");
+      return {};
+    }
+    return *quant_params->max()->cbegin();
+  }
+
+  static const float min_value(const circle::Tensor *circle_tensor)
+  {
+    const auto *quant_params = circle_tensor->quantization();
+    if (quant_params == nullptr)
+    {
+      assert(false && "There is no quantization params");
+      return {};
+    }
+    return *quant_params->min()->cbegin();
+  }
```

Comment:
for reading from the model max and min values, that were writing before executing it.